ui/app/components/scale-events-accordion.js

```diff
@@ -0,0 +1,5 @@
+import Component from '@ember/component';
+import { tagName } from '@ember-decorators/component';
+
+@tagName('')
+export default class ScaleEventsAccordion extends Component {}
```

Comment:
I think this file can be deleted entirely thanks to Octane 🤓

Reply:
OH YEAH! So if you don't make a component js file, it defaults to a glimmer component?

Reply:
ya exactly 🤠